index.html

```diff
@@ -452,6 +452,36 @@ <h2>
             </div>
             <hr>
             <!-- End template -->
+            
+            <!-- Start Template -->
+            <div class="pure-u-1 pure-u-md-1-1 pure-u-lg-1-1">
+                <img class="responsive-img"
+                    src="https://libertem.github.io/LiberTEM/_static/logo.png" />
+                <h2>
+                    LiberTEM
+                </h2>
+                <p>LiberTEM is a Python-based open source platform for high-throughput distributed processing of
+                    large-scale binary data sets such as pixelated scanning transmission electron microscopy (STEM) data.
+                    A responsive web application based on React and Redux provides a GUI, it can be used in Python scripts
+                    and Jupyter notebooks through its API, it uses dask.distributed as its default processing back-end,
+                    and it relies heavily on NumPy for processing and data handling.</p>
```

Comment:
Can we hack this maybe into separate sentences?

How about: "It can be used from Python through its API. The processing back-end uses dask.distributed by default and relies heavily on NumPy and related packages for processing and data handling. A web application based on Tornado, React, and Redux provides a GUI, in addition to using LiberTEM in combination with Jupyter notebooks."

Reply:
Nice, much better, thx!